src/AST-Core/RBMethodNode.class.st

```diff
@@ -476,6 +481,12 @@ RBMethodNode >> methodOrBlockNode [
 	"^ self"
 ]
 
+{ #category : #completion }
+RBMethodNode >> narrowWith: aString [
```

Comment:
Can you explain the meaning of narrowWith:?

Reply:
it is a bad name: better would be #updateNode: or something like that. But the whole idea to update the AST I do not like. Instead we should make sure to re-parse. I added an issue to the code completion project issue tracker for that.